src/components/Table/Table.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { fireEvent } from "@testing-library/react";
+import { Table, TableProps } from "./Table";
+import { renderCUI } from "@/utils/test-utils";
+
+const headers = [{ label: "Company" }, { label: "Contact" }, { label: "Country" }];
+
+const rows = [
+  {
+    id: "row-1",
+    items: [
+      { label: "Alfreds Futterkiste" },
+      { label: "Maria Anders" },
+      { label: "Germany" },
+    ],
+  },
+  {
+    id: "row-2",
+    items: [
+      { label: "Centro comercial Moctezuma" },
+      { label: "Francisco Chang" },
+      { label: "Mexico" },
+    ],
+  },
+];
+
+describe("Table", () => {
+  const renderTable = (props: Omit<TableProps, "headers" | "rows">) =>
```

Comment:
instead of omitting the rows and headers prop, we can just default them to the variables above so that if not pass they will get that value and if passed they can be overridden